cloudbuild.yaml

```diff
@@ -7,7 +7,7 @@ steps:
   entrypoint: bash
   env:
   - DRIVER_IMAGE_REGISTRY=us-central1-docker.pkg.dev/k8s-staging-images/dra-example-driver
-  - DRIVER_IMAGE_TAG=$COMMIT_SHA
+  - DRIVER_IMAGE_TAG=$_GIT_TAG
```

Comment:
I am unsure how expansion works in the `env` section. This might have been the problem with the original `$COMMIT_SHA`, too.

Let's do it as in https://github.com/kubernetes-csi/csi-release-tools/blob/90efb2ca59900f19eba05e65da28beda79c5bb28/cloudbuild.yaml#L32-L47 ?

Reply:
I've updated this to follow that pattern.

The [docs](https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values) show at least one example of substitutions of these automatic variables without an explicit mapping in `substitutions`. I think `COMMIT_SHA` was empty because Prow invokes a "manually triggered" build which doesn't seem to include that variable.

Reply:
Sounds good, changed this back.